en/components/overlay_styling.md

```diff
@@ -0,0 +1,200 @@
+        {{ item.name }}
+    </igx-drop-down-item>
+</igx-drop-down>
+```
+
+In our `overlay-styling.component.ts`, in addition to the `items` array we want to show, we define that our component should expose its `Host` `ElementRef` inside of the constructor:
+
+```typescript
+// overlay-styling.component.ts
+@Component({
+    selector: "app-overlay-styling",
+    styleUrls: ["overlay-styling.component.scss"],
+    templateUrl: "overlay-styling.component.html"
+})
+export class OverlayStylingComponent {
+    public items = [
+        {
+            name: "Group 1",
+            header: true,
+            id: 1
+        }, { name: "Option 1", id: 100 }, { name: "Option 2", id: 101 }, { name: "Option 3", id: 102 },
+        {
+            name: "Group 2",
+            header: true,
+            id: 2
+        }, { name: "Option 1", id: 200 }, { name: "Option 2", id: 201 }, { name: "Option 3", id: 202 }
+    ];
+
+    constructor(public element: ElementRef) {
+    }
+}
+```
+
+## Styling
+
+### Including Styles
+
+In order to use the [IgniteUI Theming Library](themes/index.md), we must first import the theme `index` file in our global styles:
+
+```scss
+@import '~igniteui-angular/lib/core/styles/themes/index';
+...
+```
+Now we can make use of all of the functions exposed by the IgniteUI theme engine.
+
+### Defining a palette
+
+After we've properly imported the index file, we create a custom palette that we can use. Let's define two colors that we like and use them to build a palette with [`igx-palette`](themes/palettes.md):
+
+```scss
+$red: #ac4365;
+$purple: #5434b8;
+
+$my-palette: igx-palette($primary: $red, $secondary: $purple);
+```
+
+### Defining themes
+
+We can now define the theme using our palette. In the example, we are using `igx-drop-down`, so we need to use [`igx-drop-down-theme`]({environment:sassApiUrl}/index.html#function-igx-drop-down-theme) to generate a theme for the component:
+
+```scss
+$my-drop-down-theme: igx-drop-down-theme(
+    $palette: $my-palette
+);
+```
+You can read more about styling the drop-down [here](drop_down.md#styling)
+
+In order to style the Overlay's modal background, we also use [`igx-overlay-theme`]({environment:sassApiUrl}/index.html#function-igx-overlay-theme):
+
+```scss
+$background-purple: #5443b84a;
+
+$my-overlay-theme: igx-overlay-theme(
+    $background-color: $background-purple
+);
+```
+
+### Applying the themes
+
+The easiest way to apply our theme is to `@include` the [`igx-drop-down`]({environment:sassApiUrl}/index.html#mixin-igx-drop-down) and [`igx-overlay`]({environment:sassApiUrl}/index.html#mixin-igx-overlay) mixins in our global styles file (typically `styles.scss`):
+
+```scss
+@include igx-overlay($my-overlay-theme)
+@include igx-drop-down($my-drop-down-theme)
+```
+
+This will apply our custom themes to **all** instances of Drop Downs and to **all** modal overlays, regardless of where they are spawned from.
+
+### Scoping Drop Down Theme
```

Comment:
Drop down should probably be left out, since this applies to everything? And scoping and emulated encapsulation are not the same thing, btw.
This should can be "Scoped component themes and view encapsulation" or something.